internal/api/v2.go

```diff
@@ -19,14 +32,51 @@ import (
 	"github.com/rundsk/dsk/internal/search"
 )
 
-func NewV2(ss *plex.Sources, appVersion string, b *bus.Broker, allowOrigins []string) *V2 {
+var (
+	//go:embed *.tmpl
```

Comment:
This is definitely nicer than what I had.

Reply:
Thanks, props to the Go standard library.